app/Http/Controllers/Staff/UploaderController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D Community Edition is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D Community Edition
+ *
+ * @author     HDVinnie <hdinnovations@protonmail.com>
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ */
+
+namespace App\Http\Controllers\Staff;
+
+use App\Http\Controllers\Controller;
+use App\Models\Group;
+use App\Models\User;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * @see \Tests\Todo\Feature\Http\Controllers\Staff\BanControllerTest
+ */
+class UploaderController extends Controller
+{
+    /**
+     * Display Stats.
+     */
+    public function index(): \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+    {
+        $uploaderGroup = cache()->rememberForever('uploader_group', fn () => Group::where('slug', '=', 'uploader')->pluck('id'));
+
+        return view('Staff.uploader.index', [
+            'uploaders' => User::with(['group'])
+                ->where('group_id', $uploaderGroup)
+                            // Count total uploads for current user
+                ->withCount(['torrents as total_uploads' => function ($query): void {
+                    $query->select(DB::raw('COUNT(*)'))
+                        ->whereColumn('user_id', 'users.id');
```

Comment:
Why are these two lines needed (I don't understand why the raw statement is needed)?

Reply:
Fixed.